### ISSUE TITLE
[iOS] Cannot type regular text on the end of the link

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor5-link#209]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

`Latest`

## 📋 Steps to reproduce

1. Place the caret on the end of the link (link is highlighted, typed text is a part f the link).
2. Try to move/place caret so the link loses highlight (on desktop you can use arrow keys) and you can type regular text on the end of the link.

## ✅ Expected result

It is possible to type regular text on the end of the link.

## ❎ Actual result

I was not able to place selection in the way to enable typing regular text on the end of the link.

TBH, I thought it is already reported by I couldn't find any similar issue 🤔 

## 📃 Other details that might be useful

Tested on iPad with iOS 12.0@Safari

**Comment by Reinmar** [comment]
Workaround for now – type as link and then unlink that part. But it'd be great if we'd find some solution for touch devices.

**Comment by oleq** [comment]
Yeah, the whole two-step-caret movement is totally keyboard-oriented. 

* Maybe we could change the way the link command works on mobiles so when at the boundary of the link, it does not unlink it all but just remove the attribute from the selection?
* Maybe we could use some gestures on mobiles? (or movement like shake, etc.)
* Maybe we could provide two-step-caret movement buttons (in the toolbar?)?
